.github/workflows/holonix-cache.yml

```
@@ -54,11 +54,17 @@ jobs:
             ${{ format(matrix.target, matrix.platform) }}
 
           # Don't exit if this fails so we can clean up the profile
+          total_status=0
           for i in result-*; do
-            cachix push holochain-ci $i || true
+            set +e
+            cachix push holochain-ci $i
+            total_status=$((total_status + $?))
```

Is `cachix push` guaranteed to always exit with a code `0` or `1`? I know it's more verbose but maybe an if-statement is safer:
```suggestion
            if ! cachix push holochain-ci $i; then
              ((total_status++))
            fi
```

or a boolean operator:
```suggestion
            cachix push holochain-ci $i || ((total_status++))
```

---

Can't hurt to track this sensibly, somebody might expect some meaning from the output. The main thing is really just that this step exits and doesn't quietly pass and leave the problem reporting to the check stage.

I couldn't get the `((total_status++))` syntax to work in Bash - but I took the suggestion with that changed to `total_status=$((total_status+1))`